ui/client/documents/index.js

```diff
@@ -639,6 +385,10 @@ const ViewDocumentsGrid = withStyles(() => ({
                 onPageChange={handlePageChange}
                 paginationMode="server"
                 rowCount={totalRowsCount}
+                disableColumnFilter
+                sortingMode="server"
+                onSortModelChange={handleSortChange}
```

Comment:
I noticed that the datagrid does not display a sort arrow when it loads (but it is sorted by `creation_date` by default). It then adds the sort arrow when explicitly selecting fields to sort (clicking on their sort arrow). We should probably display the sort arrow by default, which shouldn't harm and should inform users of the default sort order.

## Currently
#### When it first loads:

![Screenshot from 2023-05-18 10-50-38](https://github.com/jataware/dojo-stack/assets/1967061/cb2f66ac-0ad7-4830-812e-dadfb40ed6cd)


#### After manually selecting sort order:

![Screenshot from 2023-05-18 10-50-43](https://github.com/jataware/dojo-stack/assets/1967061/76691396-4492-43b8-861e-3acb48a8d1b4)

Would this cause unexpected side effects? What do you think?

Reply:
Yeah I think this is a good idea.